sdk/translation/azure-ai-translation-document/tests/test_translation.py

```diff
@@ -550,6 +550,6 @@ def test_translation_with_glossary(self, client):
 
         with open(document, "rb") as fd:
             translated = fd.readline()
-
+        # cSpell:ignore essai
```

Comment:
I recommend using an override like this in the config so we don't couple our source code to the `cspell` tool.

```
    ...
    "overrides": [
      {
        "filename": "sdk/translation/azure-ai-translation-document/tests/*.py",
        "words": [
          "essai"
        ]
      },
      ...
    ]...

```

Reply:
Got it - thanks. I somehow missed that way of ignoring while reading the docs. Opened a PR to fix here: https://github.com/Azure/azure-sdk-for-python/pull/22865